packages/cli/src/ui/utils/messageFormatting.ts

```diff
@@ -0,0 +1,156 @@
+/**
+ * @license
+ * Copyright 2025 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as path from 'path';
+
+/**
+ * Checks if a file path refers to an image based on extension
+ */
+function isImagePath(filePath: string): boolean {
+  const imageExtensions = [
```

Comment:
can you use fileUtils.ts which has a detectFileType util that already detects images?